### DISCUSSION TITLE
recipes/breads/challah/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1058] [Comments]
# recipes/breads/challah/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/challah/

**Comment by nicholaswilde** [comment]
## Toppings 
- 1 cup white chocolate chips
- 7 Tbsp blended freeze dried raspberries. 3 in dough and 4 in with chocolate.

- 1/2 cup soaked raisins kneaded in before braiding.

**Reply by nicholaswilde** [reply]
1 Tbsp raspberry powder = 6 g
